data/json/monsters/zed_ferrous.json

```
@@ -0,0 +1,278 @@
+[
+  {
+    "id": "mon_zombie_rust",
+    "type": "MONSTER",
+    "name": { "str": "rust zombie" },
+    "description": "A zombie impaled with a multitude of metal fragments that are covered with a thick layer of rust that seams to propagate to other parts of the body.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 65,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "brown",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_cut": 0,
+    "vision_night": 3,
+    "harvest": "zombie",
+    "special_attacks": [ { "type": "bite", "cooldown": 5 }, [ "GRAB", 7 ], [ "scratch", 20 ] ],
+    "attack_effs": [ { "id": "tetanus", "chance": 10 } ],
+    "death_drops": "default_zombie_death_drops",
+    "death_function": [ "NORMAL" ],
+    "burn_into": "mon_zombie_scorched",
+    "upgrades": { "half_life": 15, "into": "mon_zombie_plated" },
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_zombie_shell",
+    "type": "MONSTER",
+    "name": { "str": "rust shell zombie" },
+    "description": "This mangled body seems to develop a rusty spiked shell in its back.",
```

```suggestion
    "description": "Jagged fragments of rust-bitten metal have fused into this zombie's flesh and displaced parts of its spine.",
```
I think this zombie should get some low level armor values and cut damage so it can fit a role of small area space control and punishing melee.